benchmark/src/bin_utils.rs

```diff
@@ -77,19 +96,21 @@ pub fn measure_throughput<T: LoadGenerator + ?Sized>(
     num_epochs: u64,
 ) -> std::vec::Vec<BenchSummary> {
     // Generate testing accounts.
-    let mut accounts: Vec<AccountData> = generator.gen_accounts(num_accounts);
-    bm.register_accounts(&accounts);
-
-    // Submit minting TXNs.
-    let mint_txns = generator.gen_setup_requests(faucet_account, &mut accounts);
-    bm.mint_accounts(&mint_txns, faucet_account);
-
-    // Submit testing TXNs and measure throughput.
+    let mut accounts = gen_and_mint_accounts(bm, generator, faucet_account, num_accounts);
+    let account_chunk_size = accounts.len();
     let mut results = vec![];
     let mut throughput_seq = vec![];
     for _ in 0..num_epochs {
-        let repeated_txn_reqs = gen_repeated_requests(generator, &mut accounts, num_rounds);
-        let result = bm.measure_txn_throughput(&repeated_txn_reqs, &mut accounts, None);
+        // Submit testing TXNs and measure throughput.
+        let result = run_benchmarker_at_const_rate(
+            bm,
+            &mut accounts,
+            generator,
+            None, /* submit_rate is included in bm */
+            account_chunk_size,
+            num_rounds,
+            1, /* record result epoch by epoch */
+        );
         throughput_seq.push((result.req_throughput(), result.txn_throughput()));
```

Comment:
For later:
With summary, I'd probably skip this completely, and if printing is useful, then add something in summary itself to have a pretty printing method.

Reply:
I actually intend to impl fmt::Display for BenchSummary, but for measuring throughput, we actually want to print all epochs' results. So just a suggestion, maybe a printing method for Vec<BenchSummary>, and also printing the avg/std metrics, would be more useful.